.github/workflows/cd.yml

```
@@ -61,4 +61,16 @@ jobs:
       - name: Publish
         uses: softprops/action-gh-release@v1
         with:
-            files: 'release_*'
+          files: 'release_*'
+
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: "recursive"
+    - run: |
+       apt install python3 poetry
+       cd massa-deploy
```

The deploy script for the testnet is no longer in the massa-deploy directory.
```suggestion
       cd massa-deploy/testnet
```